internal/serve/serve.go

```diff
@@ -271,9 +279,23 @@ func initHandlerDeps(cfg Configs) (handlerDeps, error) {
 		ErrorNonJitterChannel: errorNonJitterChannel,
 		WebhookChannel:        webhookChannel,
 		TSSRouter:             router,
+		PoolPopulator:         poolPopulator,
 	}, nil
 }
 
+func populatePools(poolPopulator tssservices.PoolPopulator) {
+	alertAfter := time.Minute * 10
+	ticker := time.NewTicker(alertAfter)
+	ctx := context.Background()
+
+	for range ticker.C {
+		err := poolPopulator.PopulatePools(ctx)
+		if err != nil {
+			log.Ctx(ctx).Error("Populating pools...")
+		}
```

Comment:
We don't log the error here. I think it will be usefull for debugging.

Reply:
Done